edns.go

```diff
@@ -69,10 +71,11 @@ func (rr *OPT) String() string {
 			s += "\n; DS HASH UNDERSTOOD: " + o.String()
 		case *EDNS0_N3U:
 			s += "\n; NSEC3 HASH UNDERSTOOD: " + o.String()
-		case *EDNS0_LOCAL:
```

Comment:
Why are this file being rearranged? It doesn’t have anything to do with the constants.

Reply:
Some cleanup in anticipation of introducing `EDNS0_CHAIN` and `EDNS0_KEYTAG` types (specifically, always defining and dealing with the generic fallback type last). The changes are completely separable from introducing the new constants, though, and in fact already have their own commit. I can revert that aspect upon request if you'd rather not see it here.